### PR TITLE
Add some stuff to the README

### PR DESCRIPTION
**Pull request by unknownbreaker**
Figure it would be nice to not have a blank README.

We can always update it further, but at least it provides some basic instructions for future devs to use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+## Installation
+
+Install all packages and compile the files.
+
+```
+npm i
+npm run build
+```
+
+Create a `.env` file in the root directory of the project to store the ElephanQL database URI.
```
Comment (john-li7) [inline README.md]: Typo correction: ElephantSQL

**Review by john-li7 — COMMENTED** [review]
Reviewed, please update for typo and proceed with merge.

File: README.md (hunk 1 above)
Comment (unknownbreaker) [reply README.md]: Fixed!

Comment (unknownbreaker) [reply README.md]: Self resolving

**Comment by unknownbreaker** [comment]
@john-li7 approved for merge once typo is fixed. Merging!

**Pull request merged**